apps/src/lib/levelbuilder/CourseOfferingEditor.jsx

```diff
@@ -115,6 +133,85 @@ export default function CourseOfferingEditor(props) {
           onChange={e => updateCourseOffering('assignable', e.target.checked)}
         />
       </label>
+      <h2>{`Curriculum Catalog Settings`}</h2>
+      <label>
+        Grade Levels
+        <HelpTip>
+          <p>Select all recommended grade levels.</p>
+        </HelpTip>
+        <select
```

Comment:
Yay for a simple select! TIL that html supported this!

Reply:
🎉